### PR TITLE
[FIX] stock_barcodes: out picking location workflow

### PR DESCRIPTION
**Pull request by chienandalu**
When we've got an out picking, we're the logic workflow is that the scanned locations
is the location we're getting the stock from

cc @Tecnativa TT21753

### CODE REVIEW

**Comment by chienandalu** [comment]
@sergio-teruel Please, can you take a look?

**Review by sergio-teruel — APPROVED** [review]

File: stock_barcodes/wizard/stock_barcodes_read_picking.py [hunk 1]
```diff
@@ -83,14 +83,22 @@ def action_manual_entry(self):
         return result
 
     def _prepare_move_line_values(self, candidate_move, available_qty):
+        """When we've got an out picking, we're the logic workflow is that
```
Comment (pedrobaeza) [inline stock_barcodes/wizard/stock_barcodes_read_picking.py]: Remove `we're`
```suggestion
        """When we've got an out picking, the logic workflow is that
```

File: stock_barcodes/wizard/stock_barcodes_read_picking.py [hunk 2]
```diff
@@ -83,14 +83,22 @@ def action_manual_entry(self):
...
+           the scanned locations is the location we're getting the stock
```
Comment (pedrobaeza) [inline stock_barcodes/wizard/stock_barcodes_read_picking.py]:
```suggestion
           the scanned location is the one we're getting the stock
```

**Review by rafaelbn — APPROVED** [review]
:+1:

**Comment by chienandalu** [comment]
@pedrobaeza Changes done

**Comment by pedrobaeza** [comment]
To v12!

**Comment by chienandalu** [comment]
FW Port https://github.com/OCA/stock-logistics-barcode/pull/245

**Pull request merged**